pairsamtools/pairsam_multifilter.py

```diff
@@ -0,0 +1,470 @@
+    default="", 
+    help='output file for statistics of multiple interactors. '
+        ' If file exists, it will be open in the append mode.'
+        ' If the path ends with .gz or .lz4, the output is pbgzip-/lz4c-compressed.'
+        ' By default, statistics are not printed.')
+@click.option(
+    "--max-mismatch",
+    type=int, 
+    default=500,
+    help='This distance in (bp) from either end of a pair is used to count interacting pairs.')
+@click.option(
+    "--max-interactions",
+    type=int, 
+    default=8,
+    help='Number of maximum number of allowable interacting pairs. Interactions are counted when either position of a pair maps to "max-mismatch" distance from another pair position. ')
+@click.option(
+    '--method',
+    type=click.Choice(['max']),
+    default="max",  
+    help='define the mismatch as either the max or the sum of the mismatches of'
+        'the genomic locations of the both sides of the two compared molecules',
+    show_default=True,)
+@click.option(
+    "--sep",
+    type=str, 
+    default=_pairsam_format.PAIRSAM_SEP_ESCAPE, 
+    help=r"Separator (\t, \v, etc. characters are "
+          "supported, pass them in quotes) ")
+@click.option(
+    "--comment-char", 
+    type=str, 
+    default="#", 
+    help="The first character of comment lines")
+@click.option(
+    "--send-header-to", 
+    type=click.Choice(['low-frequency-interactors', 'high-frequency-interactors', 'both', 'none']),
+    default="both", 
+    help="Which of the outputs should receive header and comment lines")
+@click.option(
+    "--c1", 
+    type=int, 
+    default=_pairsam_format.COL_C1,  
+    help='Chrom 1 column; default {}'.format(_pairsam_format.COL_C1))
+@click.option(
+    "--c2", 
+    type=int, 
+    default=_pairsam_format.COL_C2,  
+    help='Chrom 2 column; default {}'.format(_pairsam_format.COL_C2))
+@click.option(
+    "--p1", 
+    type=int, 
+    default=_pairsam_format.COL_P1,  
+    help='Position 1 column; default {}'.format(_pairsam_format.COL_P1))
+@click.option(
+    "--p2", 
+    type=int, 
+    default=_pairsam_format.COL_P2,  
+    help='Position 2 column; default {}'.format(_pairsam_format.COL_P2))
+@click.option(
+    "--s1", 
+    type=int, 
+    default=_pairsam_format.COL_S1,  
+    help='Strand 1 column; default {}'.format(_pairsam_format.COL_S1))
+@click.option(
+    "--s2", 
+    type=int, 
+    default=_pairsam_format.COL_S2,  
+    help='Strand 2 column; default {}'.format(_pairsam_format.COL_S2))
+@click.option(
+    "--unmapped-chrom", 
+    type=str, 
+    default=_pairsam_format.UNMAPPED_CHROM,  
+    help='Placeholder for a chromosome on an unmapped side; default {}'.format(_pairsam_format.UNMAPPED_CHROM))
+@click.option(
+    "--mark-multi", 
+    is_flag=True,
+    help='If specified, duplicate pairs are marked as FF in "pair_type" and '
+         'as a duplicate in the sam entries.')
+
+@common_io_options
+
+def multifilter(pairsam_path, output, output_high_frequency_interactors,
```

Comment:
Perhaps `freqfilter` or something else? "multi" suggests having something to do with multimappers